src/Controls/tests/TestCases.HostApp/Issues/Issue28986.xaml

```diff
@@ -0,0 +1,139 @@
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue28986"
+             x:Name="TestPage"
+             Title="SafeAreaGuides Test">
+    
+    <Grid x:Name="MainGrid" BackgroundColor="Red" SafeAreaGuides.IgnoreSafeArea="All">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        
+        <!-- Content area with colored background to show safe area effects -->
+        <Grid Grid.Row="0" x:Name="ContentGrid">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            
+            <!-- Top test area - should move in/out of notch area -->
+            <Border Grid.Row="0" BackgroundColor="Yellow" Stroke="Orange" StrokeThickness="3" Padding="10">
+                <Label Text="TOP TEST AREA: This should move in/out of notch/status bar area when Top safe area is toggled"
+                       FontSize="14"
+                       FontAttributes="Bold"
+                       HorizontalOptions="Center"
+                       TextColor="DarkOrange" />
+            </Border>
+            
+            <!-- Middle content area -->
+            <Border Grid.Row="1" BackgroundColor="LightBlue" Stroke="Blue" StrokeThickness="4" Margin="10">
+                <VerticalStackLayout Spacing="15" Padding="20" VerticalOptions="Center">
+                    <Label Text="SafeAreaGuides Test Page" 
+                           FontSize="24" 
+                           FontAttributes="Bold"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue" />
+                    
+                    <Label Text="The red background shows areas outside safe area. Watch the yellow areas at top/bottom move when you toggle safe area settings."
+                           FontSize="14"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue" />
+                    
+                    <Label x:Name="CurrentSettingsLabel"
+                           Text="Current: All edges respect safe area"
+                           FontSize="16"
+                           FontAttributes="Bold"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue"
+                           AutomationId="CurrentSettings" />
+                </VerticalStackLayout>
+            </Border>
+            
+            <!-- Bottom test area - should move in/out of home indicator area -->
+            <Border Grid.Row="2" BackgroundColor="Yellow" Stroke="Orange" StrokeThickness="3" Padding="10">
+                <Label Text="BOTTOM TEST AREA: This should move in/out of home indicator/toolbar area when Bottom safe area is toggled"
```

Comment:
This border needs to be on the bottom of the view in order to test the safe areas. 

Currently the Control Panel grid is on the bottom of the screen which means as the bottom edge is enabled and disabled it doesn't have any effect on `ContentGrid`